### DISCUSSION TITLE
openwrt出现这个后阿里云盘插件会关闭，不确定问题所在

### DISCUSSION THREAD

**Discussion by tianiue** [messense/aliyundrive-webdav#206] [Q&A]
### 问题描述

启动大概一个星期左右，会一直跑这个，网卡了，重启后阿里云盘掉线，其他插件都正常，怀疑此插件引起

### 重现步骤

启动一星期左右，或者下载大文件

### 版本

1.0.0

### 运行平台

x86，lean源码openwrt

### 日志

![3BB076CA1ACD9C3AD8468C3FF538A99E](https://user-images.githubusercontent.com/61725402/146343917-e5ad4b04-a75c-4185-af89-d850b2bedfea.jpg)

**Comment by messense** [comment]
内存给的太少了？

**Comment by messense** [comment]
可以观察一下下载大文件时候的内存占用，是否被 OOM kill，`dmesg`

**Comment by messense** [comment]
搜索 blk_update_request operation 出来的结果大部分都是磁盘相关的问题，可以看看磁盘是否有问题。

**Comment by tianiue** [comment]
esxi安装的，虚拟盘1G
